File: CSharpAdvancedTopics/02.PrimeChecker/PrimeChecker.cs
using System;

class PrimeChecker
{

    static bool isPrime(int num)
    {
        int res = 0;

        for (int i = 1; i < 100; i++)
        {
            if (num % i == 0)
                res++;
        }

        return res > 2 ? false : true;
    }

    static void Main()
    {
        int n = int.Parse(Console.ReadLine());
        Console.WriteLine("isPrime? : {0}", isPrime(n));
    }
}

File: JavaScript Basics/02.JavaScript Sintax/04.Prime Checker/primeChecker.js
function isPrime (num) {
	var times = 0;
	for (var i = 0; i < num +1 ; i++) {
		if (Number(num) % i == 0){
			times++;
		}
	}

	if (times != 2){
		return false;
	}

	return true;
}

console.log(isPrime(7));
console.log(isPrime(254));
console.log(isPrime(587));
console.log(isPrime(461));
console.log(isPrime(758));
console.log(isPrime(881));